src/components/interfaces/MOBILE_API.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" standalone="no"?>
 <?xml-stylesheet type="text/xml" href="protocol2html.xsl"?>
 
-<interface name="Ford Sync RAPI" version="4.4.0" date="2017-04-27">
+<interface name="Ford Sync RAPI" version="4.4" date="2017-04-27">

```

Comment:
@LuxoftAKutsan done in https://github.com/LuxoftAKutsan/sdl_core/pull/113/commits/ab9adc7c6996e68e910941d6e3ed04f751e49215